zone/gm_commands/spawnstatus.cpp

```diff
@@ -2,27 +2,41 @@
 
 void command_spawnstatus(Client *c, const Seperator *sep)
 {
-	if ((sep->arg[1][0] == 'e') | (sep->arg[1][0] == 'E')) {
-		// show only enabled spawns
-		zone->ShowEnabledSpawnStatus(c);
```

Comment:
If the goal is to truly clean code up. All of these methods that are referenced as `Zone` scoped methods are only referenced by this GM command file. They could be inlined within the GM command file and removed from Zone.

* ShowEnabledSpawnStatus
* ShowDisabledSpawnStatus
* SpawnStatus
* ShowSpawnStatusByID

This should reduce code bloat in Zone and keep it specific to this command

Reply:
Will consolidate them all in to the GM file and remove all the methods from zone.cpp/zone.h. Will just use || inside the loop to determine how to handle each type.